src/lib/middleware/__tests__/__mocks__/request-ip.ts

```diff
@@ -0,0 +1,6 @@
+// tslint:disable-next-line:no-default-export
+export default {
+  getClientIp(_req: any) {
+    return "127.0.0.1"
+  },
+}
```

Comment:
Not sure if this is part of your struggle with Jest mocks, but I iirc mocks for packages in node_modules should be in the root `__mocks__` dir.